client/src/components/RelatedItems/AddOutfit.jsx

```diff
@@ -1,12 +1,15 @@
 import React from 'react';
+import addIcone from './images/addIcone.png';
```

Comment:
I'm being nit-picky here -- Icon is spelled w/o an 'e' at the end. Consistency is most important --- so I'd say no need to change it

Reply:
Thanks for catching this, @aaronfinsrud 🙂
I didn't realize that I put the word in Portuguese 😅
I changed to the English version.

Reply:
Thanks for catching this, @aaronfinsrud 🙂
I didn't realize that I put the word in Portuguese 😅
I changed it to the English version.